astroid/context.py

```diff
@@ -12,12 +12,34 @@
 
 """Various context related utilities, including inference and call contexts."""
 import contextlib
+import enum
 import pprint
 from typing import TYPE_CHECKING, MutableMapping, Optional, Sequence, Tuple
 
 if TYPE_CHECKING:
     from astroid.node_classes import NodeNG
 
+__all__ = [
+    "Load",
+    "Store",
+    "Del",
+    "Context",
+    "CallContext",
+    "InferenceContext",
+    "bind_context_to_node",
+    "copy_context",
+]
+
+
+class Context(enum.Enum):
+    Load = 1
+    Store = 2
+    Del = 3
+
+
+Load: Context = Context.Load
+Store: Context = Context.Store
+Del: Context = Context.Del
```

Comment:
I don't think `astroid/context.py` is the right file for it. True `Context` and `context`, but as I understand it the file mostly contains stuff regarding the `InferenceContext`. `const.py` would probably be better for it. Btw. what do you think about renaming `constants.py` to `const.py`?

Reply:
I put that in contants originally so yes I agree 😄